lib/engine/round/base.rb

```diff
@@ -235,6 +235,21 @@ def can_gain?(share, entity)
         corporation.holding_ok?(entity, share.percent) &&
         (!corporation.counts_for_limit || entity.num_certs < @game.cert_limit)
       end
+
+      def liquid_bundle?(bundle, entity)
+        return false unless @game.turn > 1
+
+        can_bundle_fit_in_bank?(bundle) && bundle.can_entity_dump?(entity)
+      end
+
+      def can_bundle_fit_in_bank?(bundle)
```

Comment:
Now that this function exists, `can_sell?` in `operating.rb` and `stock.rb` should call it instead of having their own inline copies.